packages/frint/src/App.ts

```diff
@@ -183,6 +199,10 @@ export class App {
     return this.getOption('name');
   }
 
+  public getMethods() {
```

Comment:
is this new method necessary?

Reply:
I think so if for some reason we change the way we save and get those methods, we don't need to change the implementation. Exposing this clear api we avoid future maintainability problems. What do you think?

Reply:
I am trying to convince myself to keep this, but not yet there :D 

at the moment, it is returning the original `methods` option that is passed to the App's constructor. those functions are not bound to the `app` instance, and cannot be called directly either. so don't know how they can be useful right away.

Reply:
I got your point, i thought it was about the way we are exposing this but i agree now that maybe we don't need this method.

Reply:
Maybe it would be useful for someone who wants to retrieve a list of the public methods exposed by a given app. But yes, not having the methods bound to the app instance makes the return value less useful—and it's accessible via `app.getOption('methods')` anyway.

Reply:
yep :)

Reply:
I'll remove it for now to avoid confusion. We can write a proposal later on if necessary 🙂

Reply:
But i don't know, i think we are doing some magic here, seems like a bad smell to me, we now are binding those methods and changing the scope, so we are exposing app instance inside each function, is that the desired behavior?

What do you guys think?

```javascript
const FooApp = createApp({
  name: 'Foo',
  methods: { // specify what methods this app exposes
    bar() {
      this.methods = {}; // exposing app variable
      console.log('bar')
    }
  }
})
```